### PR TITLE
Implement desktop menu redesign

### LINKED ISSUE #1370: Implement desktop menu redesign
### Overview
Our designers have redesigned the desktop menu design in #1157, so we need a frontend engineer to implement it.

### Action Items
- [ ] Take a look at the [figma](https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=1009%3A1004) and implement the changes

### Resources/Instructions
This fits in nicely with Edwin's work on switching to a light theme in #1362, so Edwin has agreed to take a look at it.

- https://github.com/hackforla/311-data/issues/1396

### LINKED ISSUE #1397: Selected information box on map has really large font
### Overview
@ajmachado42 reported:

- [ ] I'm using Firefox and the formatting in the selected information box has really large font.

image

### Resources
- https://github.com/hackforla/311-data/issues/1396

### LINKED ISSUE #1398: Would like a clear way to reset the NC selection
@ajmachado42 reported:

- [ ] There isn't a clear way to reset the NC selection--hitting exit and clicking on somewhere like the ocean doesn't work which was what I intuitively thought to do.

image

### PR DESCRIPTION
**Pull request by edwinjue**
Hi @hqasmei,

Thanks for taking a look. There is a lot here. If you need this separated into smaller PRs, please let me know and I will spend some time separating things out. For now, this is everything we are looking to merge for the v2 desktop menu redesign.

@bryantran04 copying you on this in case you have any review comments. Thanks!

Screenshots:
image
image

Resolves #1397 
Resolves #1398
Resolves #1370

  - [ ] Up to date with `dev` branch
  - [ ] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

File: client/components/main/Desktop/CouncilSelector/index.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 import React, { useState, useEffect } from 'react';
```
Comment (hqasmei) [inline client/components/main/Desktop/CouncilSelector/index.js:1]: Should Remove useState if not being used.

File: client/components/main/Desktop/CouncilSelector/index.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import { updateNcId } from '@reducers/filters';
+import { connect, useDispatch, useSelector } from 'react-redux';
```
Comment (hqasmei) [inline client/components/main/Desktop/CouncilSelector/index.js:3]: Should Remove useDispatch and useSelect if not being used.

File: client/components/main/Desktop/CouncilSelector/index.js (hunk 1 above)
Comment (hqasmei) [inline client/components/main/Desktop/CouncilSelector/index.js:1]: Should Remove useState if not being used.

File: client/components/main/Desktop/CouncilSelector/index.js [hunk 3]
```diff
@@ -21,40 +25,62 @@ const useStyles = makeStyles(theme => ({
 
 const CouncilSelector = ({
   councils,
+  selected,
+  unselected,
   updateCouncilsFilter,
+  updateSelectedCouncils,
```
Comment (hqasmei) [inline client/components/main/Desktop/CouncilSelector/index.js:31]: updateSelectedCouncils was declared in the imports section, is it a different updateSelectedCouncils?

File: client/components/main/Desktop/CouncilSelector/index.js [hunk 4]
```diff
@@ -21,40 +25,62 @@ const useStyles = makeStyles(theme => ({
...
+  updateUnselectedCouncils,
```
Comment (hqasmei) [inline client/components/main/Desktop/CouncilSelector/index.js:32]: updateUnselectedCouncils was declared in the imports section, is it a different updateUnselectedCouncils?

**Review by hqasmei — CHANGES_REQUESTED** [review]
Hey Edwin, I took a look at the changes. When you submit a PR there's a tab called checks that shows the files that need fixing to pass the automated github actions. I left comments on the places it showed would fail. Lmk if you have any questions.

File: client/components/main/Desktop/CouncilSelector/index.js (hunk 2 above)
Comment (edwinjue) [reply client/components/main/Desktop/CouncilSelector/index.js]: Thanks. Please try again. Everything should be updated.

**Comment by edwinjue** [comment]
Thanks @hqasmei. Should be good to go now

**Review by hqasmei — APPROVED** [review]

**Pull request merged**